src/Api/Service/Sf.php

```diff
@@ -526,6 +546,18 @@ public function lex_projectDto()
         return LexProjectDto::encode($this->projectId);
     }
 
+	public function lex_stats($projectCode)
```

Comment:
I guess I'm OK with this for the moment, but this is horribly inefficient to call the lexdbedto(encode) for every page load.  The method was designed to send all dictionary data to the client to be stored locally.  We should talk through our offline use case and also begin to develop a strategy where we make use of the existing indexeddb data stored locally on the client already.

I see this addition as a best effort by you to get the data you were asked to display in the dashboard while going the SSR route and I was essentially unavailable for consult or development help.

Reply:
ok, sounds like we need to talk about that more.  Putting a strategy in place to replicate the db locally and keep deltas in sync across both sources is quite an undertaking...

Reply:
yeah, we need to talk more about strategy, but this won't block the PR from being merged.